openshiftcontrolplane/v1/types.go

```diff
@@ -210,6 +210,10 @@ type ServiceAccountControllerConfig struct {
 	// managedNames is a list of service account names that will be auto-created in every namespace.
 	// If no names are specified, the ServiceAccountsController will not be started.
 	ManagedNames []string `json:"managedNames"`
+
+	// internalRegistryHostname sets the hostname for the default internal image
+	// registry. The value must be in "hostname[:port]" format.
+	InternalRegistryHostname string `json:"internalRegistryHostname"`
```

Comment:
This is specifically for the `openshift.io/serviceaccount-pull-secrets` controller right?  That should be a different stanza `ServiceAccountPullSecretsConfig`.  This stanza is for `openshift.io/serviceaccount`

Reply:
is that 
```
type DockerPullSecretControllerConfig struct {
	// registryURLs is a list of urls that the docker pull secrets should be valid for.
	RegistryURLs []string `json:"registryURLs"`
}
```

?

Reply:
(if so i guess i can just use that field and we can close this PR)

Reply:
@deads2k ok, added a new field.  ptal.